src/gaurav/SwitchCase.java

```diff
@@ -0,0 +1,29 @@
+package gaurav;
+
+public class SwitchCase {
+
+	void m1(char ch) {
+		switch (ch) {
+		case 'd':
```

Comment:
this is not right approach @GauravBhagwat1 ,
Let say tomorrow someone said we need to handle 'H', 'P' as well then again you have change you code.